circle.yml

```diff
@@ -13,6 +13,8 @@ checkout:
 
 dependencies:
   override:
+    - pip install -U pip
+    - pip install -U setuptools
```

Comment:
at this stage, I just run pip install cloudify, because you don't need any of the other packages for blueprint validation.

Reply:
Yeah, that's a good point. I wondered why we needed any of the other packages.